tools/esp8266/hmRadio.h

```diff
@@ -166,6 +166,28 @@ class HmRadio {
             return mTxChLst[mTxChIdx];
         }*/
 
+        void sendControlPacket(uint64_t invId, uint16_t data) {
```

Comment:
The function name is not that optimal. We will have more ControlPackages in the future (ON/OFF/RESET/RESTART). This name should be dedicated for power limiting. In my fork i implemented already a function call for these: https://github.com/a-marcel/ahoy/blob/main/tools/esp8266/hmRadio.h#L221

Reply:
@a-marcel Thanks for the input. My intention was /is to be as close to the implementation as it is done on the original DTU. There is that kind of function call but of course, extended with the parameter 'subcmd' aka: on/off/reset/activepowercontro/reactivepowercontrol/powerfactor which all use the same main cmd "devcontrol"
But as stated before this merge request is closed due to missing code quality / functionality